lib/import_export/import_export.rb

```diff
@@ -1,3 +1,4 @@
+require "import_export/categories_exporter"
```

Comment:
Forgot to require `categories_importer`? (when you change class name, make sure to change file name as well suitably.)